src/components/CompanionArea.js

```diff
@@ -18,19 +18,34 @@ export class CompanionArea extends Component {
     return (position === 'bottom' || position === 'far-bottom') ? classes.horizontal : null;
   }
 
+  /** */
+  collapseIcon() {
+    const { companionAreaOpen, direction } = this.props;
+    if (companionAreaOpen) {
+      if (direction === 'ltr') {
+        return <ArrowLeftIcon />;
+      }
+      return <ArrowRightIcon />;
+    }
+    if (direction === 'rtl') return <ArrowLeftIcon />;
+    return <ArrowRightIcon />;
+  }
```

Comment:
Is this style inflicted on us by the style checker, or could we make these parallel (at the very least not using both `ltr` and `rtl`.. but maybe checking  `direction` before openness would help?)?
(Assuming the style checker hates unnecessary `else`, maybe like:`) 
```suggestion
  collapseIcon() {
    const { companionAreaOpen, direction } = this.props;
    if (direction  === 'rtl') {
      if (companionAreaOpen) return <ArrowRightIcon />;
      return <ArrowLeftIcon />;
    }
    
    if (companionAreaOpen) return < ArrowLeftIcon />;
    return < ArrowRightIcon />;
  }
```

 Or would it be better to extract some kind of `CollapseIcon` that could encapsulate it? (Seems overkill but..)